assets/javascripts/discourse/connectors/above-main-container/custom-topic-lists-banner.gjs

```diff
@@ -18,10 +20,25 @@ export default class CustomTopicListsBanner extends Component {
 
   <template>
     {{#if this.customTopic}}
-      <div class="custom-list-banner banner-color">
-        <div class="custom-list-banner-contents">
-          <h1>{{this.customTopic.name}}</h1>
-          <div>{{htmlSafe this.customTopic.description}}</div>
+      {{bodyClass "category-header"}}
```

Comment:
I'm wondering if the className itself should be more robust, something more specific to make sure it is clear the meaning.

Reply:
I guess is because it uses this name in the plugin... https://github.com/discourse/discourse-category-banners/blob/eb3b1308dd7b8c5c60b89db10b2ecdf0822a8440/javascripts/discourse/components/category-banner.js#L143

Reply:
Just like @Grubba27 identified, I basically copied from the category banners.

But I certainly yield to both of your recommendations.